doc/classes/SkeletonModifier3D.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="SkeletonModifier3D" inherits="Node3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A Node that may modify Skeleton3D's bone.
+	</brief_description>
+	<description>
+		[SkeletonModifier3D] retrieves a target [Skeleton3D] by having a [Skeleton3D] parent.
+		If there is [AnimationMixer], [method SkeletonModifier3D._process_modification] always performs after playback process of the [AnimationMixer].
+		You need to execute [method Skeleton3D.get_bone_pose] and [method Skeleton3D.set_bone_pose] in [method SkeletonModifier3D._process_modification], which will be performed by the update process that [Skeleton3D] will call in deferred.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="_process_modification" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				This method can be registered with [Skeleton3D]. You should do things like get the [Skeleton3D]'s current pose and apply the pose here.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="active" type="bool" setter="set_active" getter="is_active" default="true">
+			If [code]true[/code], the [SkeletonModifier3D] will be processing.
+		</member>
```

Comment:
It is confusing that "Active" and "Visible" are separate properties here.

if SkeletonModifier3D is a child of Node3D, it seems to me that it should use the Node3D visible property instead.

I am unsure why SkeletonModifier3D is a Node3D. If it's a Node, then a custom active property makes more sense, similar to AnimationMixer.

Reply:
This is because XRModifiers inherit Node3D. I believe it make sense for SkeletonModifier3D extending Node3D in order to access the 3D coordinates of the world from within the modifier.

It may indeed be possible to integrate this with Visible since it is Node3D unlike AnimationPlayer, but I am not sure if that is a good idea.